nengo_loihi/loihi_api.py

```diff
@@ -309,10 +311,13 @@ def __init__(self, decayV, decayU, refractDelay, enableNoise):
         self.enableNoise = enableNoise
 
     def validate(self, core=None):
-        assert 0 <= self.decayU <= self.DECAY_U_MAX
-        assert 0 <= self.decayV <= self.DECAY_V_MAX
-        assert 1 <= self.refractDelay <= self.REFRACT_DELAY_MAX
-        assert 0 <= self.enableNoise <= 1
+        assert self.decayU >= 0
+        assert self.decayU <= self.DECAY_U_MAX
+        assert self.decayV >= 0
+        assert self.decayV <= self.DECAY_V_MAX
+        assert self.refractDelay >= 1
+        assert self.refractDelay <= self.REFRACT_DELAY_MAX
+        assert self.enableNoise in (0, 1)
```

Comment:
Out of curiosity, why do we have to use 0/1 for this instead of True/False?  Something to do with the NxSDK interface?

Reply:
Yeah, their interface uses 0/1, so I just used the same. We could use True/False in ours and then convert, since that would be clearer.